atlas/Snakefile

```diff
@@ -171,32 +173,101 @@ elif config.get("workflow", "complete") == "complete":
             expand("{sample}/assembly/contig_stats/final_contig_stats.txt",
                 sample=SAMPLES),
             expand("{sample}/{sample}_annotations.txt", sample=SAMPLES),
+            expand("{sample}/annotation/eggNOG.tsv", sample=SAMPLES),
             REPORTS
 
     include: "rules/assemble.snakefile"
     include: "rules/qc.snakefile"
+    include: "rules/gene_annotation.snakefile"
+
+# TODO: we should add --eggnog to the 2 merge_sample_tables and integrate it correctly in the function
```

Comment:
@brwnj Do you think you can addapt the function 
merge tables, so that it merges also the eggnogg tables? You understand better the function.

Reply:
Sure! Any chance you could email me or attach small portions of each file type? I would like to verify the headers and test any changes I make. I think all I would need is the refseq and eggnog tsv outputs, but if you have checkM output that may also be useful.

Reply:
I need the tables above updated to their latest versions (or a working example dataset). It looks like some effort has been made in `get_contigs_from_gene_names` to link Prodigal gene IDs back to their origin contig, but I'd like to see some sample output.